todocore/src/main/java/com/knighttodo/todocore/rest/DayTodoResource.java

```diff
@@ -175,4 +177,32 @@ public DayTodoReadyResponseDto updateIsReady(@PathVariable UUID dayId, @PathVari
             throw new DayTodoReadyCanNotBeUpdatedException("Day todo ready can't be updated.", ex);
         }
     }
+
+    @PatchMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiOperation(value = "Change orderNumber id existed Todo", response = DayTodoResponseDto.class, responseContainer = "List")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Created"),
+            @ApiResponse(code = 400, message = "Invalid operation"),
+            @ApiResponse(code = 403, message = "Operation forbidden"),
+            @ApiResponse(code = 500, message = "Unexpected error")
+    })
+    public List<DayTodoResponseDto> changeOrderNumberDayTodo(@PathVariable Map<UUID,Integer> map ) {
```

Comment:
you shouldn't name Map map, List list or Set set, it doesn't add meaning

Reply:
Consider using another name, changeOrderNumberInDayTodo, changeDayTodoOrderNumber but it's up to you

Reply:
Your name is better.